source/_integrations/thread.markdown

```diff
@@ -14,30 +14,144 @@ ha_platforms:
+
+Often, Thread border routing is only an auxiliary functionality of a smart home device. For example, the Nest Hub (2nd gen) is a smart display, a Google Home controller, and a Chromecast target, but also has a Thread border router included.
+
+Unlike other protocols, Thread can use multiple border routers in a single network. This increases wireless coverage and reduces the risk of a single point of failure. Ideal for home automation, with a potentially large number of devices spread over a large area.
+
+Home Assistant can only control OpenThread <abbr title="Thread border routers">TBRs</abbr>. OpenThread is an open source implementation of Thread, originally released by Google.
+
+### List of Thread border router devices
+
+Currently, the following <abbr title="Thread border router">TBR</abbr> devices are known to work well with Home Assistant:
+
+#### Nabu Casa
+
+- [Home Assistant Yellow](/yellow/) hub
+- [Home Assistant SkyConnect](/skyconnect/) Zigbee/Thread USB stick
+
+#### Google
+
+- **Displays**: Nest Hub (2nd gen), Nest Hub Max
+- **Wi-Fi routers**: Nest Wifi Pro (Wi-Fi 6E), Nest Wifi
+
+#### Apple
+
+- **Speakers**: HomePod (2nd generation), HomePod mini
+- **TVs**: Apple TV 4K (3rd generation) Wi-Fi + Ethernet, Apple TV 4K (2nd generation)
+
+#### Others
+
+There are other companies that provide devices with border router capability, such as Nanoleaf or Amazon. They won't be mentioned further because in our development and testing we focused on the above products.
+
+## Understanding the Thread configuration page
+
+This section explains why you might see multiple networks on the Thread configuration page and what this means for your network.
+
+### About different Thread networks
+
+Today, each vendor forms their own Thread network when you start using their products. This means you can end up having a Home Assistant, an Apple, and a Google Thread network in your home. The Thread configuration panel lists all Thread border routers and groups them by the Thread network.
+
+<p class='img'><img width="400" src='/images/integrations/thread/thread-no-3rd-party-credentials.png'>
+The Thread configuration page shows three vendor-specific Thread networks.
+</p>
+
+These are all separate networks using different credentials, which prevents devices from roaming between the Thread networks.
+
+The <img width="30px" src='/images/integrations/thread/information-outline.png'> icon indicates that Home Assistant has the credentials for that network. In this case, only the credentials of the `home assistant` network are known.
+
+Technical detail: Home Assistant discovers all Thread border routers in your network by their mDNS/DNS-SD announcements. The local announcements allow us to learn which networks exist, but do not contain the network credentials.
```

Comment:
```suggestion
Home Assistant discovers all Thread border routers in your network by their mDNS/DNS-SD announcements. The local announcements allow us to learn which networks exist, but do not contain the network credentials.
```

Reply:
I like this disclaimer. It tells me it's for nerds. I can skip this. I feel like to a non-technical reader, this information is almost a bit useless.

Reply:
Nice, LGTM.